### PR TITLE
Document External dependencies

### PR DESCRIPTION
**Pull request by emileswarts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,6 +224,22 @@ Note, this will only specifically enable the `RequestLogger`.
 - [Updating diagrams](/docs/updating-diagrams.md)
 - [Setting up a CircleCI context for deployment](/docs/setting-up-circleci-context-for-deployment.md)
 
+## External dependencies
```
Comment (mcoffey-mt) [inline README.md:227]:
```suggestion
### External dependencies
```
Should this be living under the documentation heading?

File: README.md [hunk 2]
```diff
@@ -223,6 +224,22 @@ Note, this will only specifically enable the `RequestLogger`.
...
+### This solution is dependent on
+- NOMIS database via the "Prison API"
+- Prisoner Offender Search
+- Probation Offender Search
+- HMPPS Auth
+- Cloud Platform
+- Azure Application Insights
+- Sentry
```
Comment (mcoffey-mt) [inline README.md:235]: > Cloud Platform
> Azure Application Insights
> Sentry

I wonder if we depend on these or just use these? Could it be worth adding another heading which might be something like "This solution uses..."

**Review by mcoffey-mt — APPROVED** [review]
Looks good! I added one or two optional comments :)

File: README.md (hunk 1 above)
Comment (emileswarts) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -44,6 +45,20 @@ as well as the following technologies for its infrastructure:
...
 ![Context Diagram](docs/diagrams/context.svg)
 
+### External dependencies
+#### This solution is dependent on
+- [NOMIS database via the Prison API](https://github.com/ministryofjustice/prison-api)
+- [Prisoner Offender Search](https://github.com/ministryofjustice/prisoner-offender-search)
+- [nDelius database via the Probation Offender Search](https://github.com/ministryofjustice/probation-offender-search)
+- [HMPPS Auth](https://github.com/ministryofjustice/hmpps-auth)
+- [Cloud Platform](https://user-guide.cloud-platform.service.justice.gov.uk/)
+- AWS in eu-west-2 (London)
```
Comment (chubberlisk) [inline README.md:55]:
```suggestion
- [Amazon Web Services (AWS)](https://aws.amazon.com/) in eu-west-2 (London)
```

File: README.md [hunk 4]
```diff
@@ -44,6 +45,20 @@ as well as the following technologies for its infrastructure:
...
+  - S3
```
Comment (chubberlisk) [inline README.md:56]:
```suggestion
  - [Simple Storage Service (S3)](https://aws.amazon.com/s3/)
```

**Review by chubberlisk — APPROVED** [review]
🚢 Just a few smol suggestions!

**Pull request merged**